package-scripts.yml

```diff
@@ -0,0 +1,48 @@
+scripts:
```

Comment:
It seems we understood issue #63 differently. What was your idea of how it would work? It shouldn't be a problem to change it so that we're all on the same page 😃